build.gradle

```diff
@@ -500,6 +500,14 @@ project('spring-integration-mail') {
 	[compileJava,compileTestJava]*.options*.compilerArgs = ["${xLintArg},-path"]
 }
 
+project('spring-integration-mock') {
```

Comment:
Ha-ha! That's true!
The end-user will still use `spring-integration-test` dependency and existing utilities will be pulled transitively from that new artifact 😄